app/components/rdf-form-fields/validation-concept-scheme-selector.js

```diff
@@ -195,4 +207,14 @@ export default class ValidationConceptSchemeSelectorComponent extends InputField
       new Statement(sourceNode, FORM('grouping'), groupingType, graph),
     ];
   }
+
+  createStatementThatConnectsPathsBetweenFieldAndValidation(
```

Comment:
This name needs some improvement..
* createStatementToConnectPaths()
* getStatementToAddFieldPathToValidation

Any idea's?